packages/cli/package.json

```diff
@@ -33,7 +33,7 @@
   },
   "scripts": {
     "test": "NODE_OPTIONS=--experimental-vm-modules npx jest --silent --coverage --runInBand --forceExit",
-    "build": "genversion --es6 src/version.ts && npx tsc -p tsconfig.json",
+    "build": "genversion --es6 src/version.ts && node ../../tools/commitHash.js && npx tsc -p tsconfig.json",
```

Comment:
I'll defer to @ukstv on if its safe to reference something outside of the package like this.  Maybe it would be safer to move this tool into the `cli` package?

Reply:
Yeah, I don't mind keeping it wherever. I put it in `tools` so it can be used from other packages as needed, but if we're only ever going to use it from the CLI, then I can move it in there.

Reply:
Actually, it was more trouble to keep it outside lol, so I moved it into the `cli` package.